base/docs/helpdb/Base.jl

```diff
@@ -881,7 +884,7 @@ trunc
 """
     unsafe_convert(T,x)
 
-Convert `x` to a value of type `T`
+Convert `x` to a C argument of type `T` where `x` is the return value of `cconvert(T, ...)`.
```

Comment:
Ah I see, you're saying there's nothing more detailed to cross-reference to then? My interpretation or suggestion is to explain the intended usage as another sentence after the sort-of-obvious ```Convert `x` to a C argument of type `T` ```. The "where" phrasing doesn't make that restriction very clear to me.